src/control/run_go_tests.sh

```diff
@@ -117,18 +117,48 @@ function setup_environment()
 	fi
 }
 
+function emit_junit_failure()
```

Comment:
Our `sh` coding style generally puts the open `{` on the same line as the function definition.

Reply:
Do you want me to change all of the other functions in this file?

Reply:
Not necessary now that this is ready for landing.  Maybe next time this file is edited if you remember.